docs/guide/object-factory.md

```diff
@@ -169,6 +172,41 @@ FireExtinguisher {
 
 - `enablePhysics`: Defines whether the fire extinguisher should have physics.
 
+## Manhole
+
+### SquareManhole
+
+A simple square manhole.
+
+%figure
+
+![SquareManhole](images/objects/manhole/SquareManhole/model.png)
+
+%end
+
+Derived from [Solid](../reference/solid.md).
+
+```
+SquareManhole {
+  SFVec3f    translation           0 0 0
+  SFRotation rotation              0 1 0 0
+  SFString   name                  "manhole"
+  SFVec3f    size                  1 0.1 1
+  SFBool     enableBoundingObject  TRUE
+}
+```
+
+> **File location**: "WEBOTS\_HOME/projects/objects/factory/manhole/protos/SquareManhole.proto"
+
+> **License**: Copyright Cyberbotics Ltd. Licensed for use only with Webots.
+[More information.](https://cyberbotics.com/webots_assets_license)
+
+#### SquareManhole Field Summary
+
+- `size`: Defines the size of the manhole.
+
+- `enableBoundingObject`: Defines whether the fire extinguisher should have physics.
```

Comment:
fire extinguisher? physics?

Reply:
Oups... I copy pasted the PROTO and forget to update this field description.